pymatgen/command_line/bader_caller.py

```diff
@@ -40,10 +38,6 @@
 __date__ = "4/5/13"
 
 
-@requires(which("bader") or which("bader.exe"),
```

Comment:
I wouldn't remove this, it's a useful error message for people who haven't installed the bader binary. I think `@requires(which(...))` probably gave an error when you added the classmethod below, but you should be able to move the `@requires()` to annotate the class `__init__` instead?